lib/puppet/functions/deprecation.rb

```diff
@@ -8,15 +8,19 @@
   end
 
   def deprecation(key, message)
+    stacktrace = Puppet::Pops::PuppetStack.stacktrace()
```

Comment:
@DavidS I'm sorry I keep doing this to you, but ...

`Evaluation Error: Error while evaluating a Function Call, uninitialized constant Puppet::Pops::PuppetStack`
https://travis-ci.org/voxpupuli/puppet-gluster/jobs/177770318#L917

That's puppet 3 with future parser enabled.